res/xml/kurdish.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+  <row>
+    <key shift="0.1665"/>
+    <key key0="ق" key1="ٯ" key2="١" key3="esc"/>
+    <key key0="و" key1="وو" key2="٢" key3="\@"/>
+    <key key0="ە" key1="ة" key2="٣" key3="\#" />
+    <key key0="ر" key1="ڕ" key2="٤" key3="$"/>
+    <key key0="ت" key1="ط" key2="٥" key3="٪"/>
+    <key key0="ی" key1="ي" key2="٦" key3="^"/>
+    <key key0="ێ" key1="ؽ" key2="٧" key3="&amp;"/>
+    <key key0="ئ" key1="ء" key2="٨" key3="*"/>
+    <key key0="ۆ" key1="ؤ" key2="٩" key3=")" key4="("/>
+    <key key0="پ" key1="ث" key2="٠"/>
+  </row>
+  <row>
+    <key shift="0.1665"/>
+    <key key0="ا" key1="أ" key2="tab"/>
```

Comment:
Same as the first row.
```suggestion
    <key shift="0.1665" key0="ا" key1="أ" key2="tab"/>
```

Reply:
My language is RTL we don't need/use the `Shift` button to uppers & lower case!

Reply:
This is not the `Shift` button but an option to add some space on the left of the keyboard.

Reply:
Oh that's great so we are now ready to merge it :)  

I tried to build an debug apk file with Android studio but i doesn't sync with gradle it looks like not the `Java` language code!

Reply:
Alright, I'll merge! 

I have not tried using Android Studio but if you find a way to make it work, I'd be happy to merge it. The project currently uses a Makefile.
I don't want to use gradle for reproducibility purposes.